src/views/Institutions/InstitutionComponents/MoUs.jsx

```diff
@@ -1,31 +1,37 @@
-import { useMemo } from "react";
 import Table from "../../../components/content/Table";
-import { Anchor } from "../../../components/content/Utils";
 
 const MoUs = ({ mou }) => {
-  mou = mou.map((mou_file) => {
-    mou_file.mou_file.url = <Anchor text={mou_file.mou_file.url} href={mou_file.mou_file.url} />;
-    return mou_file;
-  });
+  mou = mou.map((mou_obj) => {
+    if (mou_obj.mou_file.hasOwnProperty("url")) {
+      let url = mou_obj.mou_file.url;
+      if (typeof url === "string") {
+        let file_name = url.split("/").pop();
+        mou_obj.mou_file.url = (
```

Comment:
Url was giving string on the console but on UI it was giving [Object Object]. So to get the value for the filename I added the typeof.